src/Illuminate/Foundation/Testing/Constraints/FormFieldConstraint.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Illuminate\Foundation\Testing\Constraints;
+
+use Symfony\Component\DomCrawler\Crawler;
+
+abstract class FormFieldConstraint extends PageConstraint
+{
+    /**
+     * @var string
+     */
+    protected $selector;
+
+    /**
+     * @var string|bool
+     */
+    protected $value;
+
+    /**
+     * FormFieldConstraint constructor.
+     *
+     * @param  string  $selector element's name or ID
+     * @param  string  $value
+     */
+    public function __construct($selector, $value)
+    {
+        $this->selector = $selector;
+        $this->value = (string) $value;
```

Comment:
This is just to be extra sure that the value is a string. So even in case someone attempts to test this:

`$this->seeIsSelected('number', 0)` the test will behave like the expected: `$this->seeIsSelected('number', '0')`